### PR TITLE
German translations

### PR DESCRIPTION
**Pull request by diwako**
Translating to German on the side, not yet done, but to show there is progress going on.

- Adds German translations for ZEN

0 missing stringtable entry/entries.

### CODE REVIEW

**Comment by diwako** [comment]
All current stringtables are translated. Probably best to wait for someone to read through them as well, as I have no military background and jargon for a few of these things.

File: addons/editor/stringtable.xml [hunk 1]
```diff
@@ -3,39 +3,51 @@
...
             <English>Declutter Empty Tree</English>
+            <German>Leer Kategorie aufräumen</German>
         </Key>
         <Key ID="STR_ZEN_Editor_DeclutterEmptyTree_Description">
             <English>Declutters the empty tree by removing empty vehicles as those are available in the side specific trees</English>
+            <German>Räumt die Kategorie "Leer" auf. Leere Fahrzeuge werden versteckt, da diese in den Seiten spezifischen Kategorien vorhanden sind</German>
         </Key>
         <Key ID="STR_ZEN_Editor_UnitRadioMessages">
             <English>Unit Radio Messages</English>
+            <German>Einheiten Funkmeldungen</German>
```
Comment (oOKexOo) [inline addons/editor/stringtable.xml]: Sounds a bit weird, but don't have a better proposal yet.

File: addons/common/stringtable.xml [hunk 2]
```diff
@@ -68,105 +69,139 @@
...
             <English>Automatically adds spawned objects to all curators</English>
+            <German>Fügt automatisch gespawnte Objekte zu allen Kuratoren hinzu.</German>
         </Key>
         <Key ID="STR_ZEN_Common_DisableGearAnim">
             <English>Disable Gear Animation</English>
+            <German>Zahnrad Animation deaktivieren</German>
         </Key>
         <Key ID="STR_ZEN_Common_DisableGearAnim_Description">
             <English>Disables the "Gear" animation from playing when entering Zeus</English>
+            <German>Deaktiviert die "Zahnrat" Animation wenn man in dem Zeusmodus geht</German>
```
Comment (oOKexOo) [inline addons/common/stringtable.xml]: lol, he means the animation, when you open your inventory.

File: addons/common/stringtable.xml [hunk 3]
```diff
@@ -68,105 +69,139 @@
...
             <English>Changes the appearance of various user interface elements to a darker color theme</English>
+            <German>Ändert das Aussehen veschiedener Elemente der Benutzeroberfläche zu einem dunkleren Farbschema</German>
         </Key>
         <Key ID="STR_ZEN_Common_PreferredArsenal">
             <English>Preferred Arsenal</English>
+            <German>Bevorzugtes Arsenal</German>
         </Key>
         <Key ID="STR_ZEN_Common_PreferredArsenal_Description">
             <English>Selects the preferred arsenal used by Zeus Enhanced\nIf ACE Arsenal is not installed, will fall back to BI Virtual Arsenal</English>
+            <German>Regelt welches Arsenal von Enhanced Zeus geöffnet wird. Wenn ACE Arsenal nicht installiert ist, wird auf das Vanilla BI Virtual Arsenal zurückgegriffen</German>
```
Comment (oOKexOo) [inline addons/common/stringtable.xml]: would keep `Zeus Enhanced`

File: addons/common/stringtable.xml (hunk 2 above)
Comment (diwako) [reply addons/common/stringtable.xml]: Didn't know what meant with this, I now heard means the animation the unit does while in zeus mode. For some reason my character never does it so i was kind of confused.

But yeah good to know, gonna change this up.

File: addons/editor/stringtable.xml (hunk 1 above)
Comment (diwako) [reply addons/editor/stringtable.xml]: Maybe `Funkmeldungen von Einheiten`might be better.

Comment (oOKexOo) [reply addons/editor/stringtable.xml]: Yeah, I would go for that one.

File: addons/common/stringtable.xml (hunk 2 above)
Comment (dedmen) [reply addons/common/stringtable.xml]: Would be "Zahnrad" but it's actually "Ausrüstung"
It's the same as when you open inventory and lean forward and grapple around on your hips.

**Pull request merged**